mods/cnc/chrome/lobby-players.yaml

```diff
@@ -185,16 +185,13 @@ Container@LOBBY_PLAYER_BIN:
 							Y: 0 - 1
 							Width: 180
 							Height: 25
-						Button@KICK:
-							X: 180
-							Width: 25
+						DropDownButton@PLAYER_ACTION:
+							X: 15
+							Width: 190
 							Height: 25
-							Children:
-								Image:
-									ImageCollection: lobby-bits
-									ImageName: kick
-									X: 7
-									Y: 7
+							Font: Regular
+							Visible: false
+							Align: Right
```

Comment:
This doesn't look right:
![screen shot 2018-06-30 at 16 58 28](https://user-images.githubusercontent.com/167819/42126899-1ebe7380-7c87-11e8-8dd7-651e48cc73a7.png)